content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -156,17 +156,72 @@ in the Pod and to regular [init containers](/docs/concepts/workloads/pods/init-c
 ignore the Pod-level `restartPolicy` field: in Kubernetes, a sidecar is defined as an
 entry inside `initContainers` that has its container-level `restartPolicy` set to `Always`.
 For init containers that exit with an error, the kubelet restarts the init container if
-the Pod level `restartPolicy` is either `OnFailure` or `Always`.
+the Pod level `restartPolicy` is either `OnFailure` or `Always`:
+
+* `Always`: Automatically restarts the container after any termination.
+* `OnFailure`: Only restarts the container if it exits with an error (non-zero exit status).
+* `Never`: Does not automatically restart the terminated container.
 
 When the kubelet is handling container restarts according to the configured restart
 policy, that only applies to restarts that make replacement containers inside the
 same Pod and running on the same node. After containers in a Pod exit, the kubelet
-restarts them with an exponential back-off delay (10s, 20s, 40s, …), that is capped at
-five minutes. Once a container has executed for 10 minutes without any problems, the
-kubelet resets the restart backoff timer for that container.
+restarts them with an exponential backoff delay (10s, 20s, 40s, …), that is capped at
+300 seconds (5 minutes). Once a container has executed for 10 minutes without any
+problems, the kubelet resets the restart backoff timer for that container.
 [Sidecar containers and Pod lifecycle](/docs/concepts/workloads/pods/sidecar-containers/#sidecar-containers-and-pod-lifecycle)
 explains the behaviour of `init containers` when specify `restartpolicy` field on it.
 
+## How Pods handle problems with containers {#CrashLoopBackOff}
```

Comment:
Now that the page has a new shape, I suggest:
```suggestion
## How Pods handle problems with containers {#container-restarts}
```

and I would move this new section _before_ the **Restart policy** section.

Reply:
moving before the Restart policy section